CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(rtabmap_ros)
 
+
+set(RTABMap_INCLUDE_DIRS /usr/local/include/rtabmap-0.9/)
+set(RTABMap_LIBRARIES /usr/local/lib/librtabmap_core.so;/usr/local/lib/librtabmap_gui.so;/usr/local/lib/librtabmap_utilite.so)
+
```

Comment:
Not ideal, but this should be fine.

We should probably include the installation of rtabmap in our ansible script at https://github.com/braincorp/brainos_ansible_script